exercises/series/src/test/java/SeriesTest.java

```diff
@@ -17,145 +17,108 @@
+        Series series = new Series("9142");
+        List<List<Integer>> expected = Arrays.asList(Arrays.asList(9, 1), Arrays.asList(1, 4), Arrays.asList(4, 2));
+        List<List<Integer>> actual = series.slices(2);
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void canSliceByTwo() {
-        Series series = new Series("98273463");
-        List<List<Integer>> expected = Arrays.asList(
-                Arrays.asList(9, 8),
-                Arrays.asList(8, 2),
-                Arrays.asList(2, 7),
-                Arrays.asList(7, 3),
-                Arrays.asList(3, 4),
-                Arrays.asList(4, 6),
-                Arrays.asList(6, 3)
-        );
-        List<List<Integer>> actual = series.slices(2);
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
+    public void slicesIncludeDuplicates() {
+		Series series = new Series("777777");
+		List<List<Integer>> expected = Arrays.asList(
+				Arrays.asList(7, 7, 7), 
+				Arrays.asList(7, 7, 7), 
+				Arrays.asList(7, 7, 7), 
+				Arrays.asList(7, 7, 7)
+		);
+		List<List<Integer>> actual = series.slices(3);
+		assertNotNull(actual);
+		assertFalse(actual.isEmpty());
+		assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void canSliceByThree() {
-        Series series = new Series("01234");
-        List<List<Integer>> expected = Arrays.asList(
-                Arrays.asList(0, 1, 2),
-                Arrays.asList(1, 2, 3),
-                Arrays.asList(2, 3, 4)
-        );
-        List<List<Integer>> actual = series.slices(3);
+    public void slicesOfLongSeries() {
+        Series series = new Series("918493904243");
+        List<List<Integer>> expected = Arrays.asList(Arrays.asList(9, 1, 8, 4, 9), Arrays.asList(1, 8, 4, 9, 3),
+                Arrays.asList(8, 4, 9, 3, 9), Arrays.asList(4, 9, 3, 9, 0), Arrays.asList(9, 3, 9, 0, 4),
+                Arrays.asList(3, 9, 0, 4, 2), Arrays.asList(9, 0, 4, 2, 4), Arrays.asList(0, 4, 2, 4, 3));
+        List<List<Integer>> actual = series.slices(5);
         assertNotNull(actual);
         assertFalse(actual.isEmpty());
         assertEquals(expected, actual);
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void canSliceByThreeWithDuplicateDigits() {
-        Series series = new Series("31001");
-        List<List<Integer>> expected = Arrays.asList(
-                Arrays.asList(3, 1, 0),
-                Arrays.asList(1, 0, 0),
-                Arrays.asList(0, 0, 1)
-        );
-        List<List<Integer>> actual = series.slices(3);
-        assertNotNull(actual);
-        assertFalse(actual.isEmpty());
-        assertEquals(expected, actual);
+    public void sliceLengthIsToolarge() {
+        expectedException.expect(IllegalArgumentException.class);
```

Comment:
You can check the exception message here as well if you want? 🙂 Some exercises do and some don't, it gives the user a bit of an extra challenge and encourages good practice for error handling. What do you think @hgvanpariya @sjwarner-bp? 🙂

Reply:
Added Exception Message